meson_options.txt

```diff
@@ -8,6 +8,7 @@ option('lvfs', type : 'boolean', value : true, description : 'enable LVFS remote
 option('man', type : 'boolean', value : true, description : 'enable man pages')
 option('gudev', type : 'boolean', value : true, description : 'enable GUdev support')
 option('polkit', type: 'boolean', value : true, description : 'enable PolKit support in daemon')
+option('gnutls', type: 'boolean', value : true, description : 'enable GnuTLS support')
```

Comment:
We probably want to make sure this is explicitly added to `dependencies.xml` again.  When jcat was added the direct dependencies for development packages on gnutls got dropped.  They're going to be important again now with this change.

Reply:
We never actually removed it: https://github.com/fwupd/fwupd/blob/master/contrib/ci/dependencies.xml#L510 I'm not sure if it was laziness or deliberate, but either way LGTM :)